vic/drivers/image/src/vic_init_output.c

```diff
@@ -317,11 +317,11 @@ initialize_history_file(nc_file_struct *nc)
         dcount[1] = nc->ni_size;
     }
     else {
-        log_err("COORD_DIMS_OUT should be 1 or 2");
+        log_err("n_coord_dims should be 1 or 2");
     }
 
     // define the netcdf variable longitude
-    status = nc_def_var(nc->nc_id, options.DOMAIN_LON_VAR, NC_DOUBLE, ndims,
+    status = nc_def_var(nc->nc_id, global_domain.info.lon_var, NC_DOUBLE, ndims,
```

Comment:
This uses the coordinate names from the domain file.  This will handle regular or irregular grids without any issues.